maven-artifact/src/test/java/org/apache/maven/artifact/versioning/ComparableVersionTest.java

```diff
@@ -217,13 +217,13 @@ public void testVersionComparing() {
      */
     @Test
     public void testMng5568() {
-        String a = "6.1.0";
+        String a = "6.1H.5-beta"; // this is the unusual version string, with 'H' in the middle
         String b = "6.1.0rc3";
-        String c = "6.1H.5-beta"; // this is the unusual version string, with 'H' in the middle
+        String c = "6.1.0";
 
-        checkVersionsOrder(b, a); // classical
-        checkVersionsOrder(b, c); // now b < c, but before MNG-5568, we had b > c
-        checkVersionsOrder(a, c);
+        checkVersionsOrder(a, b); // now H < RC as of MNG-7559
```

Comment:
Local variables here obscure the intent. I almost missed what changed here. Inline them.

Reply:
why not, inlining the vars